imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
```

Comment:
If you're going modern, maybe go full modern and use a `=>` function here? :smile: